internal/singleflight/singleflight/syncmap_average.csv

```diff
@@ -0,0 +1,49 @@
+goroutine,duration,hit_rate
```

Comment:
same

Reply:
This file is deleted just before merging. I just leave it because I want to share the results.